pkg/cmd/streamsGet.go

```diff
@@ -89,6 +95,10 @@ func (cmd *StreamsGetCommand) createCobraCmd(
 	}
 
 	addStreamNameFlag(streamsGetCobraCmd, false, streamCommandValues)
+
+	formatters := streams.GetFormatterNamesAsString()
+	streamsGetCobraCmd.Flags().StringVar(&cmd.values.outputFormat, "format", "summary", "the output format of the returned streams. Supported formats are: "+formatters+".")
```

Comment:
Would be great to add tests for the `--format` flag in the command tests and in the GetStreams tests